backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/carousel/BpkCarousel.kt

```diff
@@ -0,0 +1,41 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.carousel
+
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import net.skyscanner.backpack.compose.carousel.internal.BpkCarouselImpl
+
+@Composable
+fun BpkCarousel(
+  imageContent: @Composable (BoxScope.(Int) -> Unit),
+  count: Int,
```

Comment:
Do we have these params name from Figma?

Reply:
the reason we need count here is we don't pass in an array of "images" into the composable, but to supply the composable with the imageContent callback from the pager content callback, to do so we need to let the Carousel know how many items are there.

And the reason to not supply images is we need external library (e.g. GlideImage) to load the image and this should not be part of the Backpack implementation.